README.md

```diff
@@ -7,7 +7,7 @@ A static [Gatsby](https://www.gatsbyjs.com/docs/) website for the Internet for A
 - TypeScript
 
 ## Requirements
-- Node.js
+- Node.js 10.0 
```

Comment:
Idk if we need to add specific versions to readme’s but ya this is how imagined the flow working